tensorflow_datasets/image/fruits360.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fruits-360: A dataset of images containing fruits.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import tensorflow as tf
+from tensorflow.python.framework import errors
+import tensorflow_datasets.public_api as tfds
+
+from tensorflow_datasets.core import api_utils
+
+_CITATION = """\
+@article{article,
+author = {Mureșan, Horea and Oltean, Mihai},
+year = {2018},
+month = {06},
+pages = {26-42},
+title = {Fruit recognition from images using deep learning},
+volume = {10},
+journal = {Acta Universitatis Sapientiae, Informatica},
+doi = {10.2478/ausi-2018-0002}
+}
+"""
+
+_DESCRIPTION = """\
+A large set of fruits on a white background.
+
+Note that the dataset is frequently updated independent of TensorFlow Datasets. To use a different version of the \
+dataset, create a custom `tfds.image.fruits360.Fruits360Config`. 
+
+```python
+config = tfds.image.fruits360.Fruits360Config(
+    name="2019.08.14.0",
+    ref="3a2533e",
+    num_classes=118
+)
+builder = tfds.builder("fruits360", config=config)
+```
+"""
+
+_IMAGE_SIZE = 100
+_IMAGE_SHAPE = (_IMAGE_SIZE, _IMAGE_SIZE, 3)
+
+
+class Fruits360Config(tfds.core.BuilderConfig):
+  """BuilderConfig for Fruits360."""
+
+  @api_utils.disallow_positional_args
+  def __init__(self,
+               name=None,
+               ref=None,
+               num_classes=None,
+               **kwargs):
+    """BuilderConfig for Fruits360.
+
+    Args:
+      name: The name of the
+      ref: The git reference (either commit or branch) for the dataset version (string).
+      num_classes: The number of classes present in the dataset (int).
+      **kwargs: keyword arguments forwarded to super.
+    """
+    self.url = "https://github.com/Horea94/Fruit-Images-Dataset/archive/{}.tar.gz".format(ref)
+    self.num_classes = num_classes
+    super(Fruits360Config, self).__init__(name=name, **kwargs)
+
+
+class Fruits360(tfds.core.GeneratorBasedBuilder):
+  """Fruits 360 dataset."""
+
+  VERSION = tfds.core.Version("1.0.0", experiments={tfds.core.Experiment.S3: True})
```

Comment:
you can remove this line (as you provide the VERSION in the builder config anyway)